tests/Dictionary_1.py

```diff
@@ -124,14 +124,14 @@ def testBadDictionary(self):
         p = Policy(self.getTestDictionary("conflict_occurs_policy_1.paf"))
         ve = ValidationError("Dictionary_1.py", 1, "testBadDictionary")
         d.validate(p, ve.cpp)
-        self.assert_(ve.getErrors("1to0") == ValidationError.TOO_MANY_VALUES)
-        self.assert_(ve.getErrors("2to1") == ValidationError.NOT_AN_ARRAY)
+        self.assert_(ve.getErrors("1to0") == int(ValidationError.TOO_MANY_VALUES))
+        self.assert_(ve.getErrors("2to1") == int(ValidationError.NOT_AN_ARRAY))
```

Comment:
On second thought, perhaps better to just try and remove the int conversions and wait for your merge. Should just work then.